xml_converter/src/xml_converter.cpp

```diff
@@ -96,32 +121,26 @@ void write_protobuf_file(string proto_filepath, map<string, Category>* marker_ca
         cout << "Error making ./protobins" << endl;
         throw std::error_code();
     }
-    // Converts map<string, Category> to Waypoint::Categories
+    // Converts map<string, Category> to a Waypoint::Category
     waypoint::Waypoint all_categories;
     std::set<string> proto_categories;
     for (const auto& category : *marker_categories) {
         waypoint::Category proto_category = category.second.as_protobuf();
         all_categories.add_category()->CopyFrom(proto_category);
         create_proto_categories(proto_category, &proto_categories);
     }
-    // Converts vector<Parseable*> to Waypoint::Icon and Waypoint::Trail
-    waypoint::Waypoint proto_pois;
+    // Collects a set of map ids from vector<Parseable*>
```

Comment:
`vector<Parseable*>` is a type, you are actually collecting map ids from `parsed_pois`, but you can do better then just saying "from parsed_pois" and instead include "from [[English description of what parsed pois is]]".

EG: An orange juicer note could say "Removes the pulp from the contents of harvester 6" or "Removes the pulp from the oranges" (the oranges which happen to be in harvester 6).